content/docs/getting-started/runtime.md

```diff
@@ -22,13 +22,38 @@ distributes load across multiple threads. The `threaded_scheduler` is the
 default for applications and the `basic_scheduler` is the default for tests.
 
 It's important to remember that all async tasks **must** be spawned on the
-runtime or no work will be performed.
+runtime or no work will be performed:
```

Comment:
This sentence talks about spawning tasks, but not about awaiting futures/async fns within a task. Maybe we should add something about await in here to set up for the following example?

Reply:
Some options:

>It's important to remember that all async tasks **must** be awaited, spawned or otherwise polled, or no work will be performed:

or

>Ultimately all async tasks must be given to the Tokio runtime, or no work will be performed. This is normally done by spawning it with `tokio::spawn` or by spawning something that awaits it. Since `connect` is an async function, the example below doesn't do anything — the `connect` function returns a `Future`, and by not awaiting or otherwise spawning this future, no work happens.

or

>Ultimately all asynchronous code must be polled to do any work. Polling the Future is the job of the Tokio runtime, but you must tell Tokio about the Future for this to happen. You can directly tell Tokio about the Future with the `tokio::spawn` function, but you can also use `.await` inside something Tokio already knows about.

Reply:
I like the first and last one.  This PR is a nice improvement to the docs.  Pick what you like so it's accurate, then we can merge.

Thank you so much!